vignettes/datatable-importing.Rmd

```diff
@@ -0,0 +1,152 @@
+---
+title: "Importing data.table"
+date: "`r Sys.Date()`"
+output:
+  rmarkdown::html_vignette:
+    toc: true
+    number_sections: true
```

Comment:
looks like I could avoid below ` {.unnumbered}` styles by setting this to false...